monero-outreach-docs/translations/es/storyboard/refuge_es.md

```diff
@@ -0,0 +1,94 @@
+ 
+La subversión de las autoridades federales a cooperar en la promoción de los acuerdos de intercambio equitativo es probable que disminuya, cuando haya cero posibilidades (eliminando la manipulación humana) de que funcionarios recuperen los XMR propios de los individuos. 
+
+### **Castigos arbitrarios y/o desproporcionados** 
+
+Al resultado de la disminución del comportamiento codicioso de la policía y funcionarios del gobierno, le seguirán los castigos desproporcionados (muchas veces arbitrarios). Con menos activos que reclamar y señalados de participar en actividades ilícitas, menor el potencial para gravar y la ejecución de dichos activos. 
+ 
+La imposición de fianzas excesivas, castigos crueles e inusuales y multas, están prohibidos para el federado, el estado y el gobierno local bajo la clausula de imposición de fianzas excesivas (aplicable al estado y a los gobiernos locales cobijados bajo la clausula de debido proceso) de la [Octava Enmienda a la Constitución de los Estados Unidos](https://es.wikipedia.org/wiki/Octava_Enmienda_a_la_Constituci%C3%B3n_de_los_Estados_Unidos). Sin embrago, como hemos visto de los ejemplos anteriores, la prohibición de las multas básicamente se queda corto en el entorno legal actual. 
+
+Monedas digitales no rastreables tales como Monero, ofrecen una privacidad sin precedentes a aquellos con grandes cantidades de dinero, que de una u otra manera podrían ser quitados con multas excesivas y nunca reembolsados. La adopción masiva y el uso de criptomonedas que preservan la privacidad también tienden a reducir las batallas sostenidas en la corte (ya que no habría que rastrear), batallas que los "políticamente débiles" y financieramente pobres no pueden combatir. 
+
+Monero se levanta como una torre de equidad en las cortes, llegado el caso que el individuo debiera encontrarse en una lucha por lo que por derecho le corresponde. 
+
+## **Reflexión final** 
+
+Mientras que las criptomonedas si permiten maneras, nunca antes vistas, para protegerse de las severas extralimitaciones estatales, es importante entender que no todas las monedas son compatibles con el mismo nivel de funcionalidad y protección. 
+ 
+Monero se separa de Bitcoin al expresar lo que es una verdadera moneda privada. Su enfoque en la privacidad y la fungibilidad, provee una capa extra de protección para el individuo al que su computador y monedas criptográficas pudieran ser incautados y vigilados por funcionarios del gobierno. Adicionalmente, provee una plataforma segura para que los usuarios guarden sus fondos que de otra manera pudieran caer presa de multas excesivas e inconstantes. 
+
+El cambio toma tiempo y no es de inmediato, como hemos señalado. Así mismo como los errores del decomiso civil no desaparecerán de la noche a la mañana, Monero y las monedas privadas le ofrecen refugio al sutil.

```

Comment:
+ _**asímismo**_.

Learned that one thanks to Word.

Reply:
done, me too, amazing!